eq-publisher/src/getQuestionnaire/queries.js

```diff
@@ -287,6 +289,7 @@ exports.getQuestionnaire = `
               confirmation {
                 id
                 title
+                pageDescription
```

Comment:
```suggestion
```
This line can be removed  - this file is only used for v2 Publisher, which is no longer maintained